capstone/client/src/components/ListPage.js

```diff
@@ -301,6 +301,8 @@ class ListPage extends Component {
         />
     ));
 
+    const saveButton = (this.state.userId === -1) ? <div></div> : <Button onClick={this.onSave} color="secondary" variant="contained">Save List</Button>;
```

Comment:
can just set it to null? don't need the empty div.

Reply:
will do in next PR, I already merged : (

Reply:
haha fast fingers